tests/data/appprotect/syslog.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: syslog
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: syslog 
+  template:
+    metadata:
+      labels:
+        app: syslog 
+    spec:
+      containers:
+      - name: syslog 
+        image: balabit/syslog-ng
```

Comment:
since this image isn't managed by us, would it make sense to use specific tag  to prevent from potential backward incompatible upgrades of the image?

Reply:
I can tag it with: https://hub.docker.com/layers/balabit/syslog-ng/3.28.1/images/sha256-645913591439d4fac37fa21dfade9589ac5b9b064b98033c8b3068be91fd22aa?context=explore